python/ray/air/_internal/accelerator_utils/device_manager.py

```diff
@@ -0,0 +1,38 @@
+from abc import ABC, abstractmethod
+from typing import List
+import torch
+
+
+class TorchDeviceManager(ABC):
```

Comment:
We can keep `get_accelerator_name`, `get_device_type` as staticmethods.

Reply:
Agreed, the changes have been made.